src/Magick.NET/Statistics/PerceptualHash.cs

```diff
@@ -64,10 +64,7 @@ internal bool Isvalid
     /// <param name="channel">The channel to get the has for.</param>
     /// <returns>The perceptual hash for the specified channel.</returns>
     public IChannelPerceptualHash GetChannel(PixelChannel channel)
```

Comment:
And should this not also be `IChannelPerceptualHash?`?

Reply:
I forgot to update the interfaces. Will do.

> Als a bit surprised this doesn't cause a warning?
I'm surprised too.